packages/core/src/amazonq/webview/ui/main.ts

```diff
@@ -197,6 +197,7 @@ export const createMynahUI = (
                 mynahUI.updateStore(tabID, {
                     loadingChat: true,
                     promptInputDisabledState: true,
+                    cancelButtonWhenLoading: true,
```

Comment:
This is causing the "stop" button to always appear which is a problem for `/transform` because we have our own stop button on the card in the chat message.

Reply:
![image](https://github.com/user-attachments/assets/3cbb81fc-f4c7-459f-a2a5-bc511d984acb)